package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "hb-multipage-site",
+  "version": "1.0.0",
+  "description": "This exercise will help you to have a better understanding of the correct structure of a web project.",
+  "main": "index.js",
+  "scripts": {
+    "start": "webpack-dev-server",
+    "build": "webpack"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/AlejandroVC67/hb-multipage-site.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/AlejandroVC67/hb-multipage-site/issues"
+  },
+  "homepage": "https://github.com/AlejandroVC67/hb-multipage-site#readme",
+  "devDependencies": {
+    "css-loader": "^0.28.9",
+    "extract-text-webpack-plugin": "^3.0.2",
+    "html-webpack-plugin": "^2.30.1",
+    "node-sass": "^4.7.2",
+    "sass-loader": "^6.0.6",
+    "standard": "^10.0.3",
```

Comment:
I don't see how is this being used in the project. Also, ideally you should be linting the js via Webpack.